### DISCUSSION TITLE
[Feature] Ability to add a non-precise date

### DISCUSSION THREAD

**Discussion by Jab2870** [immich-app/immich#8266] [Feature Request]
### The feature

There are a number of old photos that I'm scanning that I can work out the approximate year they were taken, but not the month, day, and obviously not time.

Currently, I've been putting everything like that as the 1st Jan, although it would be good if there were a better way to do it. 

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by magnuspyke** [comment]
Having the ability to set non-specific dates on photos would be a welcome addition to the product. This would require decoupling the image timestamp from the estimated date of the photograph. As a use case, I am working with approximately 10k+ photographs scanned into an archive, many ranging from the late 1800s. Currently all photographs are time-ordered by the timestamp from when they were scanned. Being able to organize them according to estimated year/month/etc would ease the pain of sorting through such a large dataset.

**Comment by timokeef** [comment]
I think an important part of this feature would be separating the date used for sorting from the precision of the historical date.
For scanned photographs we often know only part of the date: 1952, June 1952, circa 1952, or perhaps 1950–1953. Forcing these into a complete timestamp such as 1952-01-01 00:00 makes the timeline work, but it also creates metadata that implies more certainty than actually exists.
It would be useful if Immich could store/display a precision or confidence value alongside the date — e.g. Exact, Month, Year, Circa, Range, Unknown — while still maintaining an internal date for timeline sorting.
For example:
1952 → displayed as 1952
1952-06 → June 1952
approximate 1952 → c. 1952
range 1950–1953 → 1950–1953
This would be especially useful with batch editing of scanned albums. I might know that 50 photos are from “Summer 1963” without knowing the individual dates.
Ideally this information could also be preserved in XMP/metadata where practical rather than existing only as an Immich database value.
